app/main.py

```
@@ -25,17 +27,21 @@
-        return {"error": "Login failed"}
     try:
-        PasswordHasher().verify(chatuser[1], login_data.password)
-    except VerifyMismatchError as e:
+        cursor = sql_connection.cursor()
+        cursor.execute("SELECT user_id, password FROM `chat_users`.`users` WHERE `user_id` = ?", [login_data.user_id])
+        chatuser = cursor.fetchone()
+        if not chatuser:
+            return {"error": "Login failed"}
+        try:
+            PasswordHasher().verify(chatuser[1], login_data.password)
+        except VerifyMismatchError as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)